### PR TITLE
Blacken the code.

### PR DESCRIPTION
**Pull request by PurpleMyst**
Creating a PR because I'm not sure how much everybody's okay with this.

Tick your checkbox and if all 4 are lit we'll merge the PR:

- [x] \_\_Myst\_\_
- [x] Akuli
- [x] theelous3
- [x] Zaab1t

### CODE REVIEW

File: nettirely.py [hunk 1]
```diff
@@ -262,9 +262,7 @@ def _split_line(line):
                         f"SASL mechanism {sasl_mechanism!r} is not supported."
                     )
 
-                b64_query = base64.b64encode(query.encode("utf-8")).decode(
-                    "utf-8"
```
Comment (theelous3) [inline nettirely.py]: abhorrent lmao

**Review by theelous3 — APPROVED** [review]
Looks good.

**Comment by carlbordum** [comment]
Ticked. It's a pleasure to work with `black`

**Comment by Akuli** [comment]
I don't like this and I wouldn't run this gibberish on one of my projects, but it's not fucking things up too much and I'm fine with this.

**Review by Akuli — APPROVED** [review]

**Pull request merged**